app/models/cohort_import_row.rb

```diff
@@ -38,4 +32,11 @@ def home_educated
       school_urn == SCHOOL_URN_HOME_EDUCATED
     end
   end
+
+  def school_urn_inclusion
+    unless Location.school.exists?(urn: school_urn) ||
+             school_urn.in?([SCHOOL_URN_HOME_EDUCATED, SCHOOL_URN_UNKNOWN])
+      errors.add(:school_urn, "is not included in the list of valid URNs")
```

Comment:
Think we can still use the `:inclusion` validation message as before:

```suggestion
      errors.add(:school_urn, :inclusion)
```